src/tensor_parallel/pretrained_model.py

```diff
@@ -48,20 +50,20 @@ def __init__(
         if config is None:
             config = find_predefined_tensor_parallel_config(module.config, device_ids)
 
-        self.tensor_parallel = TensorParallel(module, device_ids, output_device, output_device_index, config)
+        self.wrapped_model = TensorParallel(module, device_ids, output_device, output_device_index, config)
```

Comment:
Why: if this model is Sharded(TensorParallel), calling it tensor_parallel can mislead future us.